pkg/stanza/operator/input/udp/udp.go

```diff
@@ -138,6 +138,12 @@ func (c Config) Build(logger *zap.SugaredLogger) (operator.Operator, error) {
 
 	if c.AsyncConfig != nil {
 		udpInput.messageQueue = make(chan messageAndAddress, c.AsyncConfig.MaxQueueLength)
+		udpInput.readBufferPool = sync.Pool{
+			New: func() interface{} {
```

Comment:
Looks like this is causing linter to fail: https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/6826832495/job/18567627335?pr=29100#step:8:267

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29101